### DISCUSSION TITLE
Webhook formatter

### DISCUSSION THREAD

**Discussion by antharuu** [directus/directus#11401] [Feature Requests]
Hello, ⁣:wave: 

I recently encountered the problem of not receiving the right json object format in my webhook. And I think that a small utility allowing to create its webhook in the form of a drag'n'drop or something like that could be really nice and very useful to allow extending directus without having to write code.

**Comment by rijkvanzanten** [comment]
Totally agree. @nickrum and myself are currently cooking something up that'll solve this exact use case as well 🌸🪱

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
